components/forecast/AvalancheTab.tsx

```diff
@@ -53,88 +52,67 @@ export const AvalancheTab: React.FunctionComponent<AvalancheTabProps> = React.me
 
   const elevationBandNames: ElevationBandNames = zone.config.elevation_band_names;
 
+  const [textColor] = useToken('colors', ['darkText']);
+  const renderHTMLProps: Partial<RenderHTMLProps> = {
+    contentWidth: windowWidth,
+    defaultTextProps: {
+      style: {
+        fontSize: 16,
+        color: textColor,
+      },
+    },
+  };
+
   return (
-    <VStack space="4" bgColor={'#f0f2f5'}>
-      <Card paddingTop={2} borderRadius={0} borderColor="white" header={<Heading>Avalanche Forecast</Heading>}>
+    <VStack space="2" bgColor={'#f0f2f5'}>
+      <Card marginTop={2} borderRadius={0} borderColor="white" header={<Heading>Avalanche Forecast</Heading>}>
         <HStack justifyContent="space-evenly" space="2">
           <VStack space="2" style={{flex: 1}}>
             <Text bold style={{textTransform: 'uppercase'}}>
               Issued
             </Text>
-            <Text>{dateToString(forecast.published_time)}</Text>
+            <Text color="lightText">{dateToString(forecast.published_time)}</Text>
           </VStack>
           <VStack space="2" style={{flex: 1}}>
             <Text bold style={{textTransform: 'uppercase'}}>
               Expires
             </Text>
-            <Text>{dateToString(forecast.expires_time)}</Text>
+            <Text color="lightText">{dateToString(forecast.expires_time)}</Text>
           </VStack>
           <VStack space="2" style={{flex: 1}}>
             <Text bold style={{textTransform: 'uppercase'}}>
               Author
             </Text>
-            <Text>{forecast.author || 'Unknown'}</Text>
+            <Text color="lightText">{forecast.author || 'Unknown'}</Text>
           </VStack>
         </HStack>
       </Card>
-      <View style={styles.bound}>
-        <AvalancheDangerIcon style={styles.icon} level={highestDangerToday} />
-        <View style={styles.content}>
-          <Text style={styles.title}>THE BOTTOM LINE</Text>
-          <RenderHTML source={{html: forecast.bottom_line}} contentWidth={windowWidth} />
-        </View>
-      </View>
-      <AvalancheDangerTable date={parseISO(forecast.published_time)} current={currentDanger} outlook={outlookDanger} elevation_band_names={elevationBandNames} />
-      <Text style={styles.heading}>Avalanche Problems</Text>
-      {forecast.forecast_avalanche_problems.map((problem, index) => (
-        <AvalancheProblemCard key={`avalanche-problem-${index}`} problem={problem} names={elevationBandNames} />
-      ))}
-      <Text style={styles.heading}>Forecast Discussion</Text>
-      <View style={styles.discussion}>
-        <RenderHTML source={{html: forecast.hazard_discussion}} contentWidth={windowWidth} />
-      </View>
-      <Text style={styles.heading}>Media</Text>
+      <Card
+        borderRadius={0}
+        borderColor="white"
+        header={
+          <HStack space={2} alignItems="center">
+            {/* TODO wrong icon here */}
```

Comment:
What's wrong about this icon?

Reply:
actually probably nothing! I was comparing it to the Figma mock which uses the "no information" icon, but we _should_ use the current conditions icon - I'll remove the comment.

Reply:
Yeah, there was a comment somewhere in the file about that